diff --git a/configure.pri b/configure.pri
index 8a65b72..81c84b2 100644
--- a/configure.pri
+++ b/configure.pri
@@ -823,6 +823,8 @@
         libloc_absolute_path = $$absolute_path($$config.rel_input.libdir, $$config.input.prefix)
     }
     config.input.liblocation_to_prefix = $$relative_path($$config.input.prefix, $$libloc_absolute_path)
+    config.qtbase.features.shared.available =
+    export(config.qtbase.features.shared.available)
 
     hostbindir_absolute_path = $$absolute_path($$config.rel_input.hostbindir, $$config.input.hostprefix)
     config.input.hostbindir_to_hostprefix = $$relative_path($$config.input.hostprefix, $$hostbindir_absolute_path)
